jmdaemon/jmdaemon/orderbookwatch.py

```diff
@@ -134,17 +139,74 @@ def on_order_cancel(self, counterparty, oid):
         finally:
             self.dblock.release()
 
+    def on_fidelity_bond_seen(self, nick, bond_type, fidelity_bond_b64data):
+        try:
+            fidelity_bond_data = base64.b64decode(fidelity_bond_b64data)
+        except ValueError as e:
+            log.debug("error parsing base64-encoding: " + repr(e) + ", ignoring")
+            return
+
+        #nick_sig + cert_sig + cert_pubkey + cert_expiry + utxo_pubkey + txid + vout + timelock
+        #72       + 72       + 33          + 2           + 33          + 32   + 4    + 4 = 252bytes
```

Comment:
Ah, here is the answer to my earlier question. So it is indeed just over 300 bytes(chars) when b64 expanded.